src/components/checkbox/checkbox.spec.ts

```diff
@@ -214,13 +213,47 @@ describe('MdCheckbox', () => {
       expect(testComponent.onCheckboxClick).toHaveBeenCalledTimes(1);
     });
 
-    it('should emit a change event when the `checked` value changes', async(() => {
+    it('should trigger the change event properly', async(() => {
```

Comment:
"Property" isn't very specific. How about "should trigger a change event only when the native input does" ?